lib/chains/pollUrlChain.js

```diff
@@ -63,7 +66,7 @@ const getComposers = data => {
 
 const pollUrlChain = (url, response) => makeChain(getComposers, {
 	type: 'pollUrl',
-	url,
+	url: validate(validators.NON_EMPTY_STRING, url, invalidInputMessage('pollUrl', 'URL')),
 	response,
```

Comment:
@andriichumak @givehug here should be some validation for `response`?

Reply:
Yes, it should be same - non-empty string.